### ISSUE TITLE
Error in INDEX:1 file in flowlib temp directory

### ISSUE THREAD

**Issue by jesseschalken** [facebook/flow#5112]
With Flow 0.57.1 or 0.57.2 I get this error trying to typecheck my project:

```
Library parse error:
C:\Users\Jesse\AppData\Local\Temp\flow\flowlib_10d7e02c\INDEX:1
  1: 101 file bom.js
         ^^^^ Unexpected identifier


Found 1 error
```

The directory `C:\Users\Jesse\AppData\Local\Temp\flow\flowlib_10d7e02c\` looks like a copy of the `lib/` directory in Flow, but it has this `INDEX:1` file added to it which Flow chokes on:

```
101 file bom.js
102 file core.js
103 file cssom.js
104 file dom.js
105 file indexeddb.js
106 file node.js
107 file react.js
108 file serviceworkers.js
109 file streams.js
```

If I downgrade to Flow 0.56.0 it works fine

```
No errors!
```

and the matching directory in `C:\Users\Jesse\AppData\Local\Temp\flow\` (most recently created one) doesn't contain the `INDEX:1` file.

![image](https://user-images.githubusercontent.com/1559204/31584073-b27e4380-b1f2-11e7-81a9-a8b998185565.png)

Running on Windows 10.0.15063.

**Comment by JCMais** [comment]
Same issue here!

**Comment by jesseschalken** [comment]
Yeah seems to be fixed in v0.57.3
